lib/kubernetes/src/equality/index.ts

```diff
@@ -368,15 +352,15 @@ export const hasClusterRoleChanged = (
   return false;
 };
 
-// isDeepStrictEquals doesn't handle the CRD schema very well and always reports
+// equals doesn't handle the CRD schema very well and always reports
```

Comment:
I guess this still holds true for `fast-deep-equal`, right @sreis ?

Reply:
Yes, this can be 1M+ object we can use `fast-deep-equal` to make it consistent but let's keep the workaround with the annotations.